tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml

```diff
@@ -1,3 +1,23 @@
+trigger:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
+    - 'js/web'
```

Comment:
I think it is not enough. The web pipeline depends on the core runtime. So, you cannot just only trigger this pipeline when js file changes. What you can do is: not trigger this pipeline if a PR only contains some unrelated changes such as DNNL EP/TensorRT/...